src/services/ipinfo-client.test.ts

```diff
@@ -0,0 +1,89 @@
+import { AxiosStatic } from 'axios';
+import { HttpIpinfoClient } from './ipinfo-client';
+import { exception } from 'react-ga';
+
+function createHttpIpinfoClient(axios: any) {
+  const axiosStatic = {
+    create: () => axios,
+  } as AxiosStatic;
+  const ipinfoClient = new HttpIpinfoClient({
+    axiosStatic,
+  });
+  return ipinfoClient;
+}
+
+describe('HttpIpinfoClient', () => {
+  it('should return the received country code', async () => {
+    // Arrange
+    const countryCode = 'FR';
+    const request = jest.fn(async () => ({ data: { country: countryCode } }));
+    const ipinfoClient = createHttpIpinfoClient({ request });
+
+    // Act
+    const result = await ipinfoClient.getCountryCode();
+
+    // Assert
+    expect(request).toBeCalledTimes(1);
+    expect(result).toBe(countryCode);
```

Comment:
maybe is better check with the string like `FR`

Reply:
Why? I already have a constant with that value. I think that is no necessary.

Reply:
if you see the other test we check with the strings. its more explicit. Maybe @andresmoschini know more about that :P

Reply:
I like this test with the constant, buet it is very opinable.

We can also use tables here, it will be better.